src/index.js

```diff
@@ -14,6 +15,11 @@ const isProduction = process.env.NODE_ENV === 'production';
 // Test route
 app.get('/', (req, res) => res.json({ status: res.statusCode, message: 'Welcome to Wolfsbane' }));
 
+// validation test route
+app.post('/user', validate('userRegister'), (req, res) => {
+  res.status(201).json({ message: 'success' });
+});
```

Comment:
Why is this in the place? I suppose we have a directory for declaring routes.

Reply:
Ok @eokwukwe, I was just using it for testing, I'll move to the route folder then.